src/lib/pmd/PmdWrapper.ts

```diff
@@ -49,21 +47,16 @@ export default class PmdWrapper extends PmdSupport {
 		const javaHome = await JreSetupManager.verifyJreSetup();
 		const command = path.join(javaHome, 'bin', 'java');
 
-		// The classpath needs PMD's lib folder. There may be redundancy with the shared classpath, but having the
-		// same JAR in the classpath twice is fine. Also note that the classpath is not wrapped in quotes like how it
-		// would be if we invoked directly through the CLI, because child_process.spawn() hates that.
```

Comment:
Can we continue to retain this information somewhere? I agree that comments can get outdated and irrelevant quickly, but as a team we decided to keep some form of information on code-level decisions made. We can certainly reevaluate this soon.

Reply:
Sure I can put it back.